app/src/main/java/org/fossasia/openevent/general/di/Modules.kt

```diff
@@ -275,4 +278,8 @@ val fragmentsModule = module {
     scope(Scopes.SEARCH_RESULTS_FRAGMENT.toString()) {
         FavoriteEventsRecyclerAdapter(get())
     }
+
+    scope(Scopes.SEARCH_LOCATION_FRAGMENT.toString()) {
+        PlaceSuggestionsAdapter(get())
+    }
```

Comment:
Don't use scopea

Reply:
All other adapters are initialized this way only by Koin . Shouldn't I use other way using koin for PlaceSuggestionAdapter?

Reply:
Don't use koin for this, just create a new object

Reply:
Done